5/JPA/LearningJava/src/main/java/com/wizeline/maven/learningjava/repository/UserRepository.java

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
Remove this interface, let's use the implementation directly

Reply:
ok